src/screens/SignInV2/index.js

```diff
@@ -194,16 +193,16 @@ const S = StyleSheet.create({
     fontSize: 12,
     width: 250,
     textAlign: 'center',
-    color: colors.gray,
+    color: COLORS.gray8,
     marginTop: 16
   },
   humanID: {
-    color: '#11243D',
+    color: COLORS.bunting,
     // fontWeight: 'bold',
     textDecorationLine: 'underline'
   },
-  btnText: {fontSize: 17, color: '#fff', fontWeight: 'bold'},
-  humen: {fontSize: 17, color: '#fff', fontWeight: '100'},
+  btnText: {fontSize: 17, color: COLORS.white, fontWeight: 'bold'},
+  humen: {fontSize: 17, color: COLORS.white, fontWeight: '100'},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, replacing hardcoded color values with constants from a theme file is a good practice. This makes it easier to manage and update color schemes in the future.
```diff
-    color: colors.gray,
+    color: COLORS.gray8,

-    color: '#11243D',
+    color: COLORS.bunting,

-    color: '#fff',
+    color: COLORS.white,

-    color: '#fff',
+    color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->